plugins/network-elements/nuage-vsp/src/com/cloud/network/element/NuageVspElement.java

```diff
@@ -447,13 +568,95 @@ protected boolean applyACLRules(Network network, List<? extends InternalIdentity
             }
 
         } catch (Exception e1) {
```

Comment:
Why are all checked and unchecked exceptions being caught here?  Why not catch the subset of expected checked exceptions?